deploy/olm-catalog/integreatly-operator/integreatly-operator-1.15.0/integreatly-operator.v1.15.0.clusterserviceversion.yaml

```diff
@@ -32,7 +32,7 @@ metadata:
     capabilities: Basic Install
     categories: Integreating greatly
     certified: "false"
-    containerImage: quay.io/integreatly/integreatly-operator:v1.14.0
+    containerImage: quay.io/integreatly/integreatly-operator:v1.15.0
```

Comment:
This probably shouldn't be in a "remove github-oauth-secret" commit, since as we discussed, we might need to revert that in the future...

Reply:
yeah, in reality the release to 1.15.0 should be it's own PR, so I've removed it altogether